tensorflow/core/platform/s3/s3_file_system.cc

```diff
@@ -292,8 +292,11 @@ std::shared_ptr<Aws::S3::S3Client> S3FileSystem::GetS3Client() {
 
   if (this->s3_client_.get() == nullptr) {
     AWSLogSystem::InitializeAWSLogging();
+    const char* aws_log_level = 
+        getenv("AWS_LOG_LEVEL") ? getenv("AWS_LOG_LEVEL") : Aws::Utils::Logging::LogLevel::Off;
 
     Aws::SDKOptions options;
+    options.loggingOptions.logLevel = aws_log_level
```

Comment:
I noticed by default if `AWS_LOG_LEVEL` does not exist, then the log will be completely off. I am wondering in case `AWS_LOG_LEVEL` does not exist, maybe we should just leave the default behavior?